packages/theming/atlas/src/theme/web/exclusion-variables.scss

```diff
@@ -0,0 +1,61 @@
+//
+// DISCLAIMER:
+// Do not remove this file or the variables included in this file to prevent Sass compilation failures.
+//
+
+//== Widget style exclusion
```

Comment:
I agree that moving `theme`'s exclusion variables into a separate file is a good idea, although I'm not convinced removing them from `Atlas_Core` is the right way to go. 

This approach where `theme` variables cannot be deleted isn't followed elsewhere - usually mx devs can adjust/modify freely, and `Atlas_Core` defaults the value therein. 

What's your thinking here?